apollo-runtime-kotlin/src/commonMain/kotlin/com/apollographql/apollo3/ApolloClient.kt

```diff
@@ -129,27 +131,21 @@ class ApolloClient private constructor(
       )
     }
 
-    /**
-     * internal because only used from tests
-     */
-    internal fun interceptors(interceptors: List<ApolloRequestInterceptor>) = apply {
```

Comment:
why we making these API non internal? Are we using it in different module now?

Reply:
We were using it from `apollo-integration-kotlin` but it wasn't strictly need as the public API could do too, I put everything back to `internal`